skimage/filters/thresholding.py

```diff
@@ -157,7 +157,7 @@ def threshold_local(image, block_size, method='gaussian', offset=0,
         Grayscale input image.
     block_size : int or sequence of int
         Odd size of pixel neighborhood which is used to calculate the
-        threshold value (e.g. 3, 5, 7, ..., 21, ...).
+        threshold value (e.g. 3, 5, 7, ..., 21, ...). Default block_size is 3.
```

Comment:
```suggestion
        threshold value (e.g. 3, 5, 7, ..., 21, ...).
```

no need to list defaults in the parameter description